javascript/ql/src/semmle/javascript/frameworks/NodeJSLib.qll

```diff
@@ -1060,6 +1063,15 @@ module NodeJSLib {
     }
   }
 
+  private class ClientRequestEventEmitter extends NodeJSEventEmitter {
```

Comment:
A bit of documentation, preferably with a source code example, would be nice.   

Otherwise LGTM (assuming the evaluation works out).

Reply:
Done